### PR TITLE
ops component described in documentation 

### LINKED ISSUE #309: ops: document functions
we need to document inline the functions a bit better and also make a list on the website 
make branch for off develop

### PR DESCRIPTION
**Pull request by mzuzek**
Fixes #309 

📌 [See HTML preview of `ops` documentation page](https://htmlpreview.github.io/?https://raw.githubusercontent.com/Pressio/pressio/ops-doc_309/docs/html/md_pages_components_ops.html)

### CODE REVIEW

File: docs/pages/components/ops.md [hunk 1]
```diff
@@ -10,7 +15,358 @@ Defined in header: `<pressio/ops.hpp>`
...
+  T& o,
+  const Traits<T>::size_type newRows,
+  const Traits<T>::size_type newCols)
+```
+- Resizes matrix `o` into `newCols` x `newRows` number of elements.
+@endparblock
+
+## Type Traits
+
+These pseudo-ops provide simple and convenient form for selected [type traits](md_pages_components_type_traits.html):
```
Comment (fnrizzi) [inline docs/pages/components/ops.md:364]: can you please remove the stuff below ? I don't want to expose this, it is just for internal use. 
I thought it was all inside `impl`

Comment (MikolajZuzek) [reply docs/pages/components/ops.md]: Section removed!

**Comment by fnrizzi** [comment]
@MikolajZuzek can you please remove `matching_extents`? then we are good!

**Comment by MikolajZuzek** [comment]
Removed `matching_extents`

**Comment by fnrizzi** [comment]
I have not merged this because the md will need to converted to rst soon , so let's leave it here for now

**Comment by fnrizzi** [comment]
closing this because superseeded by #389

**Pull request closed without merging**